react/components/ProductSummaryImage.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { path } from 'ramda'
+import {
+  CollectionBadges,
+  DiscountBadge,
+} from 'vtex.store-components'
+
+import Image from './Image'
+
+import productSummary from '../productSummary.css'
+
+const ProductSummaryImage = ({ product, showBadge, badgeText, showCollections }) => {
+  const {
+    productClusters,
+    productName: name,
+    sku: {
+      image: { imageUrl },
+    },
+  } = product
+
+  const commertialOffer = path(['sku', 'seller', 'commertialOffer'], product)
+
+  let img = (
+    <Image className={`${productSummary.image}`} alt={name} src={imageUrl} />
+  )
+
+  if (showBadge) {
+    img = (
+      <DiscountBadge
+        listPrice={commertialOffer.ListPrice}
```

Comment:
commertialOffer may be null by `path` definition. You may use `pathOr` to fix this, passing {} as a default

Reply:
Nice catch. Fixed.